_shows/21_22/sweeney_todd.md

```diff
@@ -0,0 +1,18 @@
+---
+title: NTU Drama Presents Sweeney Todd: The Demon Barber of Fleet Street!
```

Comment:
Set the title as 'Sweeney Todd: The Demon Barber of Fleet Street' (! optional!) and then `company: NTU Drama`